werkzeug/serving.py

```diff
@@ -543,7 +553,7 @@ def make_server(host=None, port=None, app=None, threaded=False, processes=1,
     elif threaded:
         return ThreadedWSGIServer(host, port, app, request_handler,
                                   passthrough_errors, ssl_context, fd=fd)
-    elif processes > 1:
+    elif processes > 1 and can_fork:
```

Comment:
ValueError or RuntimeError I'd say

On 30 August 2016 01:26:20 CEST, David Christenson <notifications@github.com> wrote:
>> @@ -543,7 +553,7 @@ def make_server(host=None, port=None, app=None,
>threaded=False, processes=1,
>>      elif threaded:
>>          return ThreadedWSGIServer(host, port, app, request_handler,
>>                                    passthrough_errors, ssl_context,
>fd=fd)
>> -    elif processes > 1:
>> +    elif processes > 1 and can_fork:
>
>@davidism Yeah, which exception is most appropriate? `OSError`?
>
>And should this logic remain here, or go inside the `ForkingWSGIServer`
>class initialization?
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/pallets/werkzeug/pull/999/files/8083865b6dbed6b815c21776e9384db1065bcd21#r76706604

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.